pep_sphinx_extensions/pep_processor/transforms/pep_footer.py

```diff
@@ -1,4 +1,4 @@
-import datetime
+import datetime as dt
```

Comment:
I'd prefer to be consistent and not use aliases here, if possible.
```suggestion
import datetime
```

Reply:
See: 
* https://discuss.python.org/t/renaming-datetime-datetime-to-datetime-datetime/26279/4?u=hugovk

I'll update the other one to match.

Reply:
Updated!